src/components/taskDetails/Details.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+import setColor from './DetailsColor';
+import classNames from './task-details.module.scss';
+
+type Props = {
+  detailType: string;
+  value: string;
+};
+
+const Details: FC<Props> = ({ detailType, value }) => {
+  return (
+    <div>
+      <span className={classNames['detail_type']}>{detailType}:</span>
+      <span
+        className={classNames['detail_value']}
```

Comment:
Can we please use classnames in the following way `className = {classNames.detailType}` instead to maintain consistency in codebase

Reply:
done, thanks for suggestion